test/performance/scheduler/default_rangespec.yaml

```diff
@@ -8,8 +8,8 @@ cmd:
   # Average value 351116.4 (+/- 0.9%), setting at +20%
   maxWallMs: 425_000
 
-  # Average value 396 mCPU (+/- 8%), setting at +25%
-  mCPU: 500
+  # Average value 396 mCPU (+/- 8%), setting at +30%
```

Comment:
I'm wondering if we should re-measure the average value before we release v0.7.0.
@mimowo @alculquicondor WDYT?

Reply:
Measuring this currently by hand is fairly time-consuming, so at some point I would like to do the script: https://github.com/kubernetes-sigs/kueue/issues/2077. 

Until then, I think we are ok with the current approach. If the thresholds remain too strong we can relax them by cherry-pick.

Reply:
That makes sense. After we implemented the script, I would recommend re-running every minor release (v0.8, v0.9...)